pyrs/dataobjects/sample_logs.py

```diff
@@ -200,3 +201,78 @@ def matching_subruns(self, subruns):
 
     def get_subrun_indices(self, subruns):
         return self._subruns.get_indices(subruns)
+
+
+class _DirectionExtents(NamedTuple):
+    min: float  # minimum value of the sample coordinate along one particular direction
+    max: float  # maximum value of the sample coordinate along one particular direction
+    delta: float  #
+
+
+class DirectionExtents(_DirectionExtents):
+    r"""class mimicking a namedtuple(min, max, delta) but with a different initialization argument"""
+
+    precision = 1.e-03  # two coordinates values differing by less that this amount are considered the same value
+
+    def __new__(cls, coordinates: List[float]):
+        min = min(coordinates)
```

Comment:
I think `coordinates.min()` and `coordinates.max()` are faster because they know what the underlying numpy object actually is. I'm not sure it will make a noticeable difference for the size arrays we are using.

Reply:
> I think `coordinates.min()` and `coordinates.max()` are faster because they know what the underlying numpy object actually is. I'm not sure it will make a noticeable difference for the size arrays we are using.

`list` objects don't have a `min()` method